database/java/2017/15095.yaml

```diff
@@ -0,0 +1,44 @@
+---
+cve: 2017-15095
+title: CVE in com.fasterxml.jackson.core:jackson-databind
+description: >
+    A deserialization flaw was discovered in the jackson-databind in versions before 2.8.10 and 2.9.1, which could allow an unauthenticated user to perform code execution by sending the maliciously crafted input to the readValue method of the ObjectMapper. This issue extends the previous flaw CVE-2017-7525 by blacklisting more classes that could be used maliciously.
+cvss_v2: 7.5
+references:
+    - http://www.securitytracker.com/id/1039769
+    - https://access.redhat.com/errata/RHSA-2017:3189
+    - https://access.redhat.com/errata/RHSA-2017:3190
+    - https://access.redhat.com/errata/RHSA-2018:0342
+    - https://github.com/FasterXML/jackson-databind/issues/1680
+    - https://github.com/FasterXML/jackson-databind/issues/1737
+    - https://security.netapp.com/advisory/ntap-20171214-0003/
+    - https://www.debian.org/security/2017/dsa-4037
+
+affected:
+    - groupId: com.fasterxml.jackson.core
+      artifactId: jackson-databind
+      version:
+        - "<=2.8.10,==2.9.1"
+      fixedin:
+        - "2.8.11.1, 2.9.4"
```

Comment:
@msrb  It was fixed twice, is this the right format for it?

Reply:
Not quite :)

If I read the description correctly, then I'd say that affected versions are:
```
      version:
        - "<=2.8.9"
        - "<=2.9.0,2.9.0.pr1"
```
I.e. all 2.9.x versions [before 2.9.1](https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind) and all versions before 2.8.10.

`fixedin` would then be:
```
      fixedin:
        - ">=2.8.10"
        - ">=2.9.1"
```